### ISSUE TITLE
GetServiceInfoListAsync not working on a 6.2.274 cluster

### ISSUE THREAD

**Issue by PepijnGramberg** [Microsoft/service-fabric-client-dotnet#7]
GetServiceInfoListAsync fails on the newest version (6.2.274) of a cluster (not sure if it worked on older versions)

The reason it fails is because when parsing the serviceInfo, the response it reads the first property and expects it to be "ServiceKind" but the response it got is something like the following:

```
{"ContinuationToken":"", "Items":[{"Id":"devApp~SomeActorService", "ServiceKind":"Stateful","...},...]}
```

**Comment by amanbha** [comment]
It's a know issue andit will be fixed in next update of Service Fabric 6.2.* and 6.3 release

**Comment by amanbha** [comment]
This should work with clusters with version >=6.2.183
